tests/utilities/test_flow_decorator.py

```diff
@@ -161,6 +152,10 @@ def test_flow_decorator_flow_in_flow(
     # WHEN Calling the _test_flow_in_flow function with specific messages
     _flow_over_flow(call_llm_messages)
 
+    # Wait for the Prompt span to be exported; It was asynchronously waiting
+    # on the OpenAI call span to finish first
+    time.sleep(1)
```

Comment:
Added some time.sleep in tests since the processor now waits async before sending the HL spans to processor